app/src/lib/rules/syntax_rules.js

```diff
@@ -120,6 +120,17 @@ const builtin_syntax_rules = [
 			}),
 		},
 	},
+	{
+		name: 'Tag words with a possessive \'s as genitive_saxon',
+		comment: '',
+		rule: {
+			trigger: token => token.type === TOKEN_TYPE.LOOKUP_WORD && /^(.+?)('s|')(-[A-Z])?$/.test(token.token),
```

Comment:
Do we care to put this regex in `regexes.js` so the naming and or docs can help understand the "why or what" of it?